documentation/docs/article/guides/rails_integration_with_null_driver.md

```diff
@@ -84,3 +84,62 @@ describe 'example', driver: :null do
   end
 end
 ```
+
+## Minitest Usage
+
+We can do something similar with the default Rails setup using Minitest. Here's the same example written with Minitest:
+
+```rb
+# test/application_system_test_case.rb
+
+require 'playwright'
+
+class CapybaraNullDriver < Capybara::Driver::Base
+  def needs_server?
+    true
+  end
+end
+
+Capybara.register_driver(:null) { CapybaraNullDriver.new }
+
+class ApplicationSystemTestCase < ActionDispatch::SystemTestCase
+  driven_by :null
+
+  def self.playwright
+    @playwright ||= Playwright.create(playwright_cli_executable_path: 'npx playwright')
```

Comment:
`npx playwright` is just for trial usage and `'./node_modules/.bin/playwright'` would be better for test usage.

Reply:
I was trying to be consistent with the rest of the docs. Since it's usually nested, then it would either be:

```rb
Rails.root.join("node_modules/.bin/playwright")

# OR

"../node_modules/.bin/playwright"
```

Reply:
I thought `./node_modules/.bin/playwright` works well on most cases, including RSpec.
Does Minitest on Rails require `../node_modules/.bin/playwright` ?

I agree that `Rails.root.join("node_modules/.bin/playwright")` would be better for Rails users.